meshroom/ui/qml/Viewer3D/MediaLibrary.qml

```diff
@@ -203,11 +203,14 @@ Entity {
                 property string rawSource: attribute ? attribute.value : model.source
                 // whether dependencies are statified (applies for output/connected input attributes only)
                 readonly property bool dependencyReady: {
-                    if(attribute && attribute.isOutput)
-                        return attribute.node.globalStatus === "SUCCESS";
-                    if(attribute && attribute.isLink)
-                        return attribute.linkParam.node.globalStatus === "SUCCESS";
-                    return true;
+                    if(!attribute)
+                        // if the node is removed, the attribute will be invalid
+                        return false
+                    if(attribute.isOutput)
+                        return attribute.node.globalStatus === "SUCCESS"
+                    if(attribute.isLink)
+                        return attribute.linkParam.node.globalStatus === "SUCCESS"
```

Comment:
linkParam is not recursive by default.
You should use: `getLinkParam(recursive=True)` (not yet exposed as a slot).
Maybe we should expose it as a property, like originLinkParam.